### PR TITLE
Allow mgr bootstrap keyring to be defined

### PR DESCRIPTION
**Pull request by ggillies**
In environments where we wish to have manual/greater control over
how the bootstrap keyrings are used, we need to able to externally
define what the mgr keyring secret will be and have ceph-ansible
use it, instead of it being autogenerated

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1610213

### CODE REVIEW

**Comment by leseb** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1610213

**Comment by ggillies** [comment]
Updated with variable placed in ```roles/ceph-mon/defaults/main.yml``` and run ```generate_group_vars_sample.sh```

**Comment by ggillies** [comment]
@leseb I updated my original commit to fix your request (I didn't want to add a new commit and dirty the commit history), how doI got about unblocking?

**Comment by guits** [comment]
@ggillies could you add the `Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1610213` in your commit message please?

**Comment by ggillies** [comment]
@leseb @guits Is there anything else that I need to change here?

Regards,

Graeme

File: roles/ceph-mon/tasks/ceph_keys.yml [hunk 1]
```diff
@@ -76,6 +76,7 @@
       osd: allow *
       mds: allow *
     cluster: "{{ cluster }}"
+    secret: "{{ (mgr_secret is defined and mgr_secret != 'mgr_secret') | ternary(mgr_secret, omit) }}"
```
Comment (leseb) [inline roles/ceph-mon/tasks/ceph_keys.yml]: `mgr_secret is defined` is not necessary since it's declared in `defaults/main.yml`

**Comment by ggillies** [comment]
@leseb fixed

**Comment by guits** [comment]
@ggillies I don't see the change, could you check your last commit?

**Comment by ggillies** [comment]
@guits ah sorry I have pushed the change up now. Unfortunately it seems CI has failed due to a vagrant issue?

**Comment by guits** [comment]
jenkins test pipeline

**Pull request merged**